tests/benchmarks/test_fields.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from random import randrange
```

Comment:
To be consistent with eleswhere can we `import random` then use `random.randrange`? The benefit is at a glance you can see where it comes from.